### PR TITLE
Wait links if we have exposed port

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace .. OPM ?

File: data/busybox/Dockerfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
 FROM busybox
-CMD sleep 60
+CMD tail -f /dev/null
```
Comment (bersace) [inline data/busybox/Dockerfile]: :+1:

File: data/link/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM busybox
+EXPOSE 1234
+CMD sleep 2 && nc -l 0.0.0.0 -p 1234 && nc -l 0.0.0.0 -p 1234 && tail -f /dev/null
```
Comment (bersace) [inline data/link/Dockerfile]: :+1:

**Comment by bersace** [comment]
Belle PR. OPM.

**Pull request merged**